Gems/EMotionFX/Code/EMotionFX/Source/AnimGraphNodeGroup.h

```diff
@@ -174,13 +174,26 @@ namespace EMotionFX
          */
         bool IsNameEditOngoing() const;
 
+        /**
+         * Returns a the id of the parent AnimGraphNode to which this group belongs.
```

Comment:
nit: Remove 'a' here so it just reads...

```
Returns the id of the parent AnimGraphNode to which this group belongs.
```

Reply:
Otherwise +1 to the excellent descriptive API comment 👍

Reply:
Done

Reply:
Thanks!